### PR TITLE
build for node 6 and node 8

### LINKED ISSUE #14: node 8
node 8 is out, but we are still pinning node 6. Need to wait for node 8 to be fixed on Windows, but that should be solved once https://github.com/conda-forge/nodejs-feedstock/pull/19 lands.

### PR DESCRIPTION
**Pull request by minrk**
put nodeX in the build string

closes #14

### CODE REVIEW

**Comment by minrk** [comment]
node 8 seems to make some symlinks that conda-build doesn't like. Will investigate what's up there.

**Comment by willingc** [comment]
@minrk I restarted the travis build and it passed. Can you restart Appveyor?

**Comment by minrk** [comment]
The appveyor failures seem to be real. I haven't had a chance to figure out what's going wrong.

**Comment by willingc** [comment]
@minrk If you can add me to the feedstock team, I can see if I spot anything on Windows.

**Comment by minrk** [comment]
@willingc great idea! Open a PR adding yourself to [the maintainers](https://github.com/conda-forge/configurable-http-proxy-feedstock/blob/master/recipe/meta.yaml#L35) list and I'll merge it. The change takes a while to propagate (it can be hours).

**Comment by willingc** [comment]
Thanks @minrk.

**Comment by minrk** [comment]
@willingc I had a look at this again and I think I might know what's up. The first thing I had to change is that `npm install -g <path>` creates a symlink instead of doing a normal install with npm 5.x. This appears to be a non-optional change (annoying). So what I ended up doing was running `npm pack` to repackage a tarball, then do `npm install -g <tarball>` to do a normal install. I think on Windows, my two-line script doesn't work so it's only doing the pack and skipping the next step. The latest commit separates the build into `bld.bat` so both lines ought to be executed now.

**Comment by willingc** [comment]
Cool. Looks great and tests pass :tada: I'll let you merge when you are done tweaking the Windows install commands.

**Comment by minrk** [comment]
Latest pass was erroneous because it didn't run on Windows. Had to poke it to try again. Hopefully it will work.

The previous build failed, apparently because it simply wasn't running more than one line of bld.bat, which is strange.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
+  fn: {{ name }}-{{ version }}.tar.gz
   url: https://registry.npmjs.org/{{ name }}/-/{{ name }}-{{ version }}.tgz
   sha256: eb41627ed15505261dbbb48eb4b9f1c6cd31a884b0f1afa9001a671b67262ad0
 
 build:
-  number: 0
-  script: npm install -g .
+  number: {{ build }}
+  string: "node{{node_major}}_{{ build}}"
+  script: npm pack; npm install -g {{name}}-{{version}}.tgz  # [not win]
```
Comment (jakirkham) [inline recipe/meta.yaml]: For multiline scripts, one can do this.

ref: https://github.com/conda-forge/waf-feedstock/blob/ad1e7be11615070c8940ff1d2ebc4d2cfd5ed408/recipe/meta.yaml#L15-L18

**Pull request merged**